lib/EarlyFunctionAnalysis/EarlyFunctionAnalysis.cpp

```diff
@@ -613,6 +616,44 @@ void FunctionAnalysisResults::importModel() {
   }
 }
 
+void FunctionAnalysisResults::importDynamicSymbols() {
+  llvm::Function &Root = *M.getFunction("root");
+
+  for (BasicBlock &BB : Root) {
+    if (getType(&BB) != BlockType::JumpTargetBlock)
+      continue;
+
+    auto *SymbolNameString = hasDynamicSymbol(&BB);
+    if (not SymbolNameString)
+      continue;
+
+    auto [Entry, _] = getPC(&*BB.begin());
+
+    // Already imported?
+    if (FunctionsBucket.find(Entry) != FunctionsBucket.end())
+      continue;
+
+    llvm::StringRef SymbolName = extractFromConstantStringPtr(SymbolNameString);
+    revng_assert(SymbolName.size() != 0);
+
+    // TODO: change this to assert, dynamic function must have been imported
```

Comment:
I don't think we'll ever want to assert this. I foresee the possibility of having a symbol associated to `newpc` even without an entry in `ImportedDynamicFunctions`.
At very least, turn this is into a question.